MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Reflectometry/IReflSettingsTabView.h

```diff
@@ -62,6 +63,34 @@ class DLLExport IReflSettingsTabView {
   virtual std::string getStitchOptions() const = 0;
   virtual void
   createStitchHints(const std::map<std::string, std::string> &hints) = 0;
+
+  /// Experiment settings
+  virtual std::string getAnalysisMode() const = 0;
+  virtual std::string getDirectBeam() const = 0;
+  virtual std::string getCRho() const = 0;
+  virtual std::string getCAlpha() const = 0;
+  virtual std::string getCAp() const = 0;
+  virtual std::string getCPp() const = 0;
+  virtual std::string getMomentumTransferLimits() const = 0;
+  virtual std::string getDetectorLimits() const = 0;
+  virtual std::string getScaleFactor() const = 0;
+  virtual std::string getPolarisationCorrections() const = 0;
+  /// Instrument settings
+  virtual std::string getMonitorIntegralMin() const = 0;
+  virtual std::string getMonitorIntegralMax() const = 0;
+  virtual std::string getMonitorBackgroundMin() const = 0;
+  virtual std::string getMonitorBackgroundMax() const = 0;
+  virtual std::string getLambdaMin() const = 0;
+  virtual std::string getLambdaMax() const = 0;
+  virtual std::string getI0MonitorIndex() const = 0;
+
+  /// Request presenter to obtain default values for settings
+  virtual void requestExpDefaults() const = 0;
+  virtual void requestInstDefaults() const = 0;
```

Comment:
I don't think we need these methods to be defined in the abstract base class. In our abstract view we only need to define methods that will be used by the concrete presenter. It is enought to define them in the concrete view, as you've done below. When removed, they should be removed from the `MockSettingsTabView` too.